src/patternfly/base/normalize.scss

```
@@ -108,12 +108,52 @@
   :where(a) {
     color: var(--pf-t--global--text--color--link--default);
     text-decoration-line: var(--pf-t--global--text-decoration--link--line--default);
+    text-decoration-style: var(--pf-t--global--text-decoration--link--style--default);
+    text-decoration-color: var(--pf-t--global--text-decoration--color--default);
+    text-underline-offset: var(--pf-t--global--text-decoration--offset--default);
+
+    // RHDS uses: text-underline-offset: max(var(--pf-t--global--text-decoration--offset--default), 0.28em);
```

Could you write it like this?

```suggestion
    text-underline-offset: var(--pf-t--global--text-decoration--offset--default); // RHDS uses: text-underline-offset: max(var(--pf-t--global--text-decoration--offset--default), 0.28em);
```